source/app/datamgmt/manage/manage_case_templates_db.py

```
@@ -309,3 +378,49 @@ def case_template_post_modifier(case: Cases, case_template_id: Union[str, int]):
+
+    case_template = CaseTemplate.query.filter_by(id=case_template_id).first()
+    case_task = get_task(task_id=task_id)
+    if not case_template:
+        return []
+
+    actions = []
+
+    for task in case_template.tasks:
+
+        if case_task.task_title == task["title"]:
+           if 'actions' in task:
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)